dll/ntdll/ldr/ldrapi.c

```diff
@@ -1098,6 +1098,7 @@ LdrQueryProcessModuleInformationEx(
     _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
     {
         /* Ignoring the exception */
+        // FIXME: It looks like STATUS_SUCCESS could be returned, though *ReturnedSize was not set, for example.
```

Comment:
I personally don't see the point of you adding FIXME's in code that you do not seem to understand fully.